### PR TITLE
[rachelhox] BDU-671

### PR DESCRIPTION
**Pull request by rachelhox**
## Checklist
- [ ] Ran Linting
- [ ] Targeted PR against correct branch.
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote unit tests.
- [ ] Added an entry to the `CHANGELOG.md` file.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

**Comment by teamchong** [comment]
BDU-672
[Mobile] Footer size amendment
image

**Comment by teamchong** [comment]
BDU-673
[Background] Size amendment 
image

**Comment by teamchong** [comment]
BDU-674
[Grid] Adjust grid
image

**Comment by teamchong** [comment]
BDU-676
[Footer] Adjust text size
image

File: src/components/SectionBox/useStyles.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ const useStyles = () => {
       justifyContent: 'center',
       minWidth: '272px',
       '&.sectionbox__main': {
-        padding: theme.spacing(3),
+        padding: theme.spacing(3,0),
```
Comment (teamchong) [inline src/components/SectionBox/useStyles.ts:12]:
```suggestion
        padding: theme.spacing(3, 0),
```
suggest to add space here, to have a consistent style

**Review by teamchong — APPROVED** [review]

**Comment by teamchong** [comment]
LGTM except one minor formatting issue

File: src/components/SectionBox/useStyles.ts (hunk 1 above)
Comment (rachelhox) [reply src/components/SectionBox/useStyles.ts:12]: @teamchong how about `theme.spacing(3, 1)`?

Comment (teamchong) [reply src/components/SectionBox/useStyles.ts:12]: all good for me, so the previous `theme.spacing(3,0)` is typo?

Comment (rachelhox) [reply src/components/SectionBox/useStyles.ts:12]: No, was focusing on the left and right padding, and the title has a top padding of 24px so thought `(3, 0)` is appropriate.

**Review by teamchong — APPROVED** [review]
LGTM

**Pull request merged**